### ISSUE TITLE
Rules are not found in release `1.1.1`

### ISSUE THREAD

**Issue by jannisg** [testing-library/eslint-plugin-jest-dom#10]
With my `.eslintrc.js` file as follows:

```js
module.exports = {
  extends: [
    'plugin:testing-library/react',
    'plugin:jest-dom/recommended',
  ],
}
```

I am getting an error (for each rule) of 

```
error  Definition for rule 'XXX' was not found
```

where `XXX` refers to all of the rules from this plugin (`prefer-checked`, `prefer-enable-disable`, and `prefer-required`)

---

Downgrading my `eslint-plugin-jest-dom` to version `1.1.0` fixes this issue and linting works as expected.

**Comment by benmonro** [comment]
@gnapse  can you look into this?  I think the auto generated thing may be causing an issue here

**Comment by gnapse** [comment]
Not sure why you ask me, as I really have no idea about the auto generated thing. A quick search in the PRs suggests we should involve @nickmccurdy in the conversation, given his contribution in #4.

**Comment by benmonro** [comment]
Oops sorry. Picked the wrong name from the drop down.  My apologies

**Comment by nickmccurdy** [comment]
What's your version of `requireindex`? You can check with `npm ls requireindex` or `yarn why requireindex`. If it's at least 1.2.0 this should work, in which case it's an issue with publishing the package.

**Comment by gnapse** [comment]
> Oops sorry. Picked the wrong name from the drop down. My apologies

No problem. Glad to be involved anyway.

**Comment by nickmccurdy** [comment]
Whoops, looks like I forgot to add the package name prefix, sorry about that. PR incoming.

**Comment by jannisg** [comment]
Thanks @nickmccurdy :) 

I have just upgraded my project to use `1.1.2` and can confirm this has been fixed. 🎉 

Different timezone so just got back into this, FWIW, my `requireindex` version was indeed at `1.2.0`.

**Comment by benmonro** [comment]
@jannisg actually 1.1.2 was a revert of 1.1.1.  1.1.3 is the actual fix, would you mind giving it a spin to ensure it's working for you?

**Comment by jannisg** [comment]
@benmonro Upgraded my project to `1.1.3` and it's all working as expected. 🎉
